app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,12 @@
+# frozen_string_literal: true
+
+require 'pry'
 class PostsController < ApplicationController
-  before_action :can_edit, only: [ :edit, :update, :destroy ]
-  before_action :check_time!, only: [:edit, :update]
+  before_action :can_edit, only: %i[edit update destroy]
```

Comment:
What does %i do?

Reply:
Rubocop change. Without it you need to write: [ :edit, :update, :destroy ] instead of %i[edit update destroy]. In other words, it allows you to use a non-interpolated array of symbols separated by whitespace!